### ISSUE TITLE
Fails to run a raspberry pi

### ISSUE THREAD

**Issue by felixgalindo** [cazala/coin-hive#74]
(node:4000) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Failed to launch chrome!
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 1: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: ELF: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 1: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: cannot open S: No such file
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 1: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �hP��PôPô����: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 1: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: @�: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 2: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: @%$@@@0ppp�c�c��c��c��c@�@�,#�,#Q�td��c��c��c: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 3: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 4: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 5: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 6: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 7: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: �: not found
/home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: 8: /home/pi/monero-miner/node_modules/puppeteer/.local-chromium/linux-497674/chrome-linux/chrome: Syntax error: ")" unexpected

TROUBLESHOOTING: https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md

(node:4000) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code

node -v
v8.9.3

uname -a
Linux raspberrypi 4.9.59-v7+ #1047 SMP Sun Oct 29 12:19:23 GMT 2017 armv7l GNU/Linux

**Comment by felixgalindo** [comment]
Here is my code:


`const CoinHive = require('coin-hive');

(async() => {

  // Create miner
 const miner = await CoinHive('wXVQDwG8Rkq7XYDn0fAgM6HcXPU9tFMQ', {
  launch: {
    executablePath: '/usr/bin/chromium-browser', 
    args: ['--disable-setuid-sandbox', '--no-sandbox']
  }
});

  // Start miner
  await miner.start();

  // Listen on events
  miner.on('found', () => console.log('Found!'))
  miner.on('accepted', () => console.log('Accepted!'))
  miner.on('update', data => console.log(`
    Hashes per second: ${data.hashesPerSecond}
    Total hashes: ${data.totalHashes}
    Accepted hashes: ${data.acceptedHashes}
  `));

  // Stop miner
  setTimeout(async() => await miner.stop(), 60000);
})();`

**Comment by cazala** [comment]
have you tried this? https://github.com/cazala/coin-hive#troubleshooting

**Comment by cazala** [comment]
specially this:

```
const miner = await CoinHive('site-key', {
  launch: {
    executablePath: '/usr/bin/chromium-browser', 
    args: ['--disable-setuid-sandbox', '--no-sandbox']
  }
});
```

**Comment by felixgalindo** [comment]
Yes I tried that. It doesn’t work because the wrong version of chromium gets installed. I had to install a different version manually and point it to it.

**Comment by cazala** [comment]
@felixgalindo so is the issue solved?

**Comment by felixgalindo** [comment]
Well I resolved it on my own using a work-around but anyone who tries to use the coin-hive library will experience the same issue.

**Comment by cazala** [comment]
if you could share the workaround so I include it in the troubleshooting/faqs section it would be very helpful (:
